master/internal/api_experiment.go

```diff
@@ -1469,6 +1469,27 @@ func (a *apiServer) parseAndMergeContinueConfig(expID int, overrideConfig string
 			fmt.Sprintf("override config must have single searcher type got '%s' instead", overrideName))
 	}
 
+	// Determine which workspace the experiment is in.
+	wkspName := activeConfig.Workspace()
+	if wkspName == "" {
+		wkspName = "Uncategorized"
```

Comment:
This isn't defined as a constant anywhere or automatically filled out already?

Reply:
Ahh it is! Just replaced this with a constant, good idea!